api/helpers.go

```diff
@@ -51,6 +52,38 @@ func (a *API) sendTx(tx []byte) (*cometcoretypes.ResultBroadcastTx, error) {
 	return resp, nil
 }
 
+func (a *API) getBlockFromAppOrIndexer(height int64) (*Block, error) {
+	if tmblock := a.vocapp.GetBlockByHeight(height); tmblock != nil {
+		return &Block{
+			Block: comettypes.Block{
+				Header:     tmblock.Header,
+				Data:       tmblock.Data,
+				Evidence:   tmblock.Evidence,
```

Comment:
Do we need evidence?

Reply:
i haven't touched this, it's just the previous code, and i'd leave it as-is.

but better yet, i've simplified this and now just use tmblock altogether

Reply:
https://github.com/vocdoni/vocdoni-node/compare/dbdd78ff508f48e60a450f43d5a5d139a50ef990..711f82e200237bdd406173d8d5874eb462b3a96a